From 559cf0cd1e226baf63a98c39572264fbf5c3f6b4 Mon Sep 17 00:00:00 2001
From: David Teigland <teigland@redhat.com>
Date: Tue, 23 Apr 2019 09:39:42 -0500
Subject: [PATCH] devices: drop open error message

This open error is being printed in more common,
non-error circumstances than expected.  After a
number of complaints make it only a debug message.
---
 lib/device/dev-io.c | 5 +----
 1 file changed, 1 insertion(+), 4 deletions(-)

diff --git a/lib/device/dev-io.c b/lib/device/dev-io.c
index 2a83a9657..6996a44dc 100644
--- a/lib/device/dev-io.c
+++ b/lib/device/dev-io.c
@@ -572,10 +572,7 @@ int dev_open_flags(struct device *dev, int flags, int direct, int quiet)
 			}
 		}
 #endif
-		if (quiet)
-			log_sys_debug("open", name);
-		else
-			log_sys_error("open", name);
+		log_sys_debug("open", name);
 
 		dev->flags |= DEV_OPEN_FAILURE;
 		return 0;
-- 
2.21.0

